README.md

```diff
@@ -40,3 +45,27 @@ mvn package
 java -jar target\jwt-grant-generator-1.0-SNAPSHOT-jar-with-dependencies.jar myclient.properties
 
 ```
+
+### Output as JSON
+If you want the response as json, you can add an additional parameter so the command to build and run is
+```
+mvn package
+
+java -jar target\jwt-grant-generator-1.0-SNAPSHOT-jar-with-dependencies.jar myclient.properties json
```

Comment:
I chose this way of selecting the output format and not as named parameter, in order to avoid pulling in another package to support that.